### PR TITLE
add fitPlant by @KrisKodira

### PR DESCRIPTION
**Pull request by KrisKodira**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

**Comment by KrisKodira** [comment]
> 	3 Warnings
> ⚠️ 	Found 2 link issues
> ⚠️ 	Found 1 link issue
> ⚠️ 	Found 8 link issues
> #### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)
> Line 	Status 	Link
> 84 	[404](https://httpstatuses.com/404) 	https://itunes.apple.com/app/g.o.-tracker-4-compassion/id1100240821
> 174 	[404](https://httpstatuses.com/404) 	https://itunes.apple.com/app/compass-a-minimal-compass-app/id1231545538
> #### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)
> Line 	Status 	Link
> 1247 	[404](https://httpstatuses.com/404) 	http://www.linphone.org/technical-corner/linphone/downloads
> #### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)
> Line 	Status 	Link
> 79 	[301](https://httpstatuses.com/301) 	https://github.com/fnzeta/WhatsUp redirects to
> https://github.com/satishbabariya/WhatsUp
> 1265 	[404](https://httpstatuses.com/404) 	https://github.com/JRock007/Tob
> 1724 	[301](https://httpstatuses.com/301) 	https://github.com/jVirus/ios-spritekit-tic-tac-toe redirects to
> https://github.com/jVirus/tic-tac-toe
> 1930 	[404](https://httpstatuses.com/404) 	https://raw.githubusercontent.com/nightscout/nightguard/master/images/screen1.jpg
> 1931 	[404](https://httpstatuses.com/404) 	https://raw.githubusercontent.com/nightscout/nightguard/master/images/screen2.jpg
> 1932 	[404](https://httpstatuses.com/404) 	https://raw.githubusercontent.com/nightscout/nightguard/master/images/screen3.jpg
> 1947 	[404](https://httpstatuses.com/404) 	https://github.com/matiasvillaverde/mobile-ios-vipergame
> 1948 	[404](https://httpstatuses.com/404) 	https://raw.githubusercontent.com/matiasvillaverde/mobile-ios-vipergame/master/header.png
> 
> Generated by 🚫 [Danger](https://danger.systems/)

Is this relevant for me?

File: contents.json [hunk 1]
```diff
@@ -6590,6 +6590,15 @@
       "date_added": "Wed, 1 Apr 2015 09:30:22 -0400",
       "suggested_by": "@jeffreyjackson"
     },
+    {
+            "title": "fitPlant",
+            "category-ids": ["fitness"],
+            "description": "Workout tracker where you can grow artificial plants",
+            "source": "https://github.com/KrisKodira/fitPlant",
+            "screenshots": ["https://github.com/KrisKodira/fitPlant/blob/master/design/ios-app/main-view.png"],
```
Comment (vpeschenkov) [inline contents.json]: @KrisKodira the link should be direct to the image. Could you fix that?

**Comment by vpeschenkov** [comment]
> Is this relevant for me?

Hi @KrisKodira, don't pay attention to that. It isn't relevant to you :-)

File: contents.json (hunk 1 above)
Comment (KrisKodira) [reply contents.json]: Fixed it :) 👍

**Comment by vpeschenkov** [comment]
@KrisKodira thank you!

**Pull request merged**